CONTRIBUTING.md

```diff
@@ -116,7 +117,7 @@ To link your Codecademy user profile to GitHub:
 
 - Before making a PR, make sure you pushed your changes from a branch other than `main`.
 - Name the new branch after the changes being pushed to the PR.
-- Keep your PRs byte-sized. No more than 3 new entries per PR! And no more than 3 PRs open at a time!
+- Keep your PRs byte-sized. No more than 3 new entries per PR! And do not open more than 3 PRs in a two week period!
```

Comment:
Would this apply if all their PRs are merged before two weeks are up? I'm thinking for cases of edits which typically are faster to merge.

Reply:
@KTom101 I changed it to not having more than three PRs open in a two week period.